sandbox/OPEN-SOURCE.md

```diff
@@ -0,0 +1,29 @@
+# Open Source
+
+Defense Unicorns values Open Source. We understand that Open Source is a way for all members of the company to prevent silos and work in a collaboration-first mentality. 
+
+## Advocates
+- Brandt Keller
+
+## Why?
+
+Open Source directly contributes to the company vision:
+
+> A free and secure world that enables progress and innovation
+
+Progress and Innovation are spaces that any member of Defense Unicorns can strive towards.
+It is also a space where we can contribute to the betterment of society through knowledge sharing and teaching others. These others may often be other Uncicorns who are baordening their horizions to new technologies. It may also be external, and the sharinf of processes, information, data and code for the purpose of building a more secure tomorrow. 
+
+## Internal Impact
+Advocating Open Source as a company value means promoting collaboration to all facets of company operation in open and traceable manners. Asynchronous collaboration and documentation allow traceability to be a forcing function for the members of the company to understand current thoughts and provide historical reference for when something needed to change. This allows all members of the company to understand and provide valuable context and perspectives that may not have been originally established.
```

Comment:
We could also add `free` in areas like this such as `...promoting collaboration and the free sharing of ideas to...`